plugins/couchbase/couchbaseembed.cpp

```diff
@@ -181,8 +181,7 @@ namespace couchbaseembed
             auto status = pQcmd->named_param(cbPlaceholder.str(), utf8);
             if (!status.success())
                 failx("Could not bind Param: %s val: %s", cbPlaceholder.str(), utf8);
```

Comment:
Not new, but this will leak the utf8 string on an exception.

We usually use a pattern like:

    rtlDataAttr utf8;
    unsigned utf8chars;
    rtlStrToUtf8X(utf8chars, utf8.refstr(), len, value);
    auto status = pQcmd->named_param(cbPlaceholder.str(), utf8.getstr());
...

Reply:
Yes, I think so, if you are changing it anyway